sql/core/src/test/resources/sql-tests/inputs/pgSQL/comments.sql

```diff
@@ -0,0 +1,48 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- COMMENTS
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/comments.sql
```

Comment:
Shall we use `BETA3` from now? I believe it will be the same with `BETA2`.
- https://github.com/postgres/postgres/blob/REL_12_BETA3/src/test/regress/sql/comments.sql

Reply:
Yes. Changed it to `REL_12_BETA3`.